.github/workflows/build_wheels.yml

```diff
@@ -119,19 +123,24 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
+    - name: Install Visual C++ for Python 2.7 on Windows
```

Comment:
cp27 is missing for windows-latests above. Why are we installing a Python 2.7 compiler?

Reply:
Good point, that should be a letftover. I've just removed it.